src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.tsx

```diff
@@ -390,15 +390,15 @@ const PopularRepositoriesTable = () => {
                         <Dropdown
                           onSelect={onDropdownSelect}
                           className={classes.addRepositoriesButton}
-                          ouiaId='add-selected-toggle-dropdown'
+                          ouiaId='add-selected-dropdown'
```

Comment:
lets revert this change to avoid partial match. The original will work fine with the other changes.